generators/app/templates/src/app/components/InputLabel/index.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function InputLabel({
+  containerClassName,
+  textClassName,
+  dataFor,
+  label,
+  inputClassName,
+  inputName,
+  placeholder,
+  inputId,
+  inputType,
+  handleChange,
+  disabled
+}) {
+  return (
+    <div className={containerClassName}>
+      <span className={`${textClassName} m-bottom-1`} data-for={dataFor}>
+        {label}
+      </span>
+      <input
+        className={inputClassName}
+        name={inputName}
+        placeholder={placeholder}
+        id={inputId}
+        type={inputType}
+        onChange={handleChange}
+        disabled={disabled}
+      />
+    </div>
+  );
+}
+
+InputLabel.propTypes = {
+  containerClassName: PropTypes.string,
+  textClassName: PropTypes.string,
+  dataFor: PropTypes.string.isRequired,
+  label: PropTypes.string.isRequired,
+  inputClassName: PropTypes.string,
+  inputName: PropTypes.string.isRequired,
+  placeholder: PropTypes.string,
+  inputId: PropTypes.string.isRequired,
+  inputType: PropTypes.string.isRequired,
+  handleChange: PropTypes.func.isRequired,
+  disabled: PropTypes.bool
+};
+
+InputLabel.defaultProps = {
+  containerClassName: 'column start',
```

Comment:
instead of putting `column start` as default props, I'd put it as the initial class value and anyone could override it. I don't like that much putting default class props because if you add anything to `containerClassName` that has nothing to do with the alignment (for example, a background-color), you also loose the alignment

```
<div className={`column start ${containerClassName}`}>

...

InputLabel.defaultProps = {
  containerClassName: '',
```